source/indexer/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@airswap/indexer",
-  "version": "1.5.7",
+  "version": "1.5.8",
```

Comment:
since deployed to mainnet update the major version.

Reply:
oh yeah....